ProcessMaker/Http/Controllers/Api/UserTokenController.php

```diff
@@ -76,6 +111,36 @@ public function index(Request $request, User $user)
      *
      * @param  \Illuminate\Http\Request  $request
      * @return \Laravel\Passport\PersonalAccessTokenResult
+     *
+     * /**
+     * Display listing of access tokens for the specified user
+     *
+     * @OA\Post(
+     *     path="/users/{user_id}/tokens",
+     *     summary="Create new token for a specific user",
+     *     operationId="createTokens",
+     *     tags={"Personal Tokens"},
+     *     @OA\Parameter(
+     *         description="User id",
+     *         in="query",
```

Comment:
@AbdelilahLbardi  Same

When testing the endpoints that user_id is being included as a query for example /users/{user_id}/tokens?user_id=2 instead of /users/2/tokens. To fix this update in="query" to be in="path".

Make sure to run php artisan l5-swagger:generate when making changes to update the api-docs.json file.

Reply:
✅  done!